perfzero/lib/benchmark.py

```diff
@@ -91,6 +91,7 @@ def _get_benchmark_methods(self):
           if re.match(pattern, benchmark_method_name):
             benchmark_methods.append(benchmark_class + '.' +
                                      benchmark_method_name)
+        del class_instance
```

Comment:
Have you tested that this actually frees up the memory?

Reply:
I have tested that the instructions in the README to verify that the test works. I have not verified that this actually frees up the memory. I will remove this change from the patch.